tests/models/test_hooks.py

```diff
@@ -574,7 +574,114 @@ def training_step(self, batch, batch_idx):
     assert called == expected
 
 
-def test_trainer_model_hook_system_fit_no_val_and_resume(tmpdir):
+def test_trainer_model_hook_system_fit_no_val_and_resume_max_epochs(tmpdir):
+    # initial training to get a checkpoint
+    model = BoringModel()
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_epochs=1,
+        limit_train_batches=2,
+        limit_val_batches=0,
+        enable_progress_bar=False,
+        enable_model_summary=False,
+        callbacks=[HookedCallback([])],
+    )
+    trainer.fit(model)
+    best_model_path = trainer.checkpoint_callback.best_model_path
+
+    called = []
+    callback = HookedCallback(called)
+    # already performed 1 step, resume and do 2 more
```

Comment:
Can you explain the comment? I think it's wrong.

We first train 2 steps (1 epoch). 
Then, we resume to do 2 more steps (1 additional epoch)

Reply:
yep... it's a small typo... must be copied from the test right below